ogr/services/gitlab/project.py

```diff
@@ -360,6 +360,13 @@ def change_token(self, new_token: str):
     def get_branches(self) -> list[str]:
         return [branch.name for branch in self.gitlab_repo.branches.list(all=True)]
 
+    def get_commits(self, ref: Optional[str] = None) -> list[str]:
+        ref = ref or self.default_branch
+        return [
+            commit.id
+            for commit in self.gitlab_repo.commits.list(ref_name=ref, all=True)
```

Comment:
I have a feeling that the `list` was a not the best decision _waaaay back in the old days_.